symfit/tests/tests_ode.py

```diff
@@ -105,8 +99,8 @@ def test_simple_kinetics(self):
         a0 = 54 * 10e-4
 
         model_dict = {
-            D(a, t): - k * a**2,
-            D(b, t): k * a**2,
+            Derivative(a, t): - k * a**2,
```

Comment:
Your namespacing is ~~a mess~~ complicated ;)

from symfit import D:
ImportError: cannot import name 'D'

from symfit.api import D:
ImportError: cannot import name 'D'

from symfit.support import D:
ImportError: No module named 'symfit.support'

I finally found the answer:
from symfit.core.support import D
But renaming D to Derivative was the faster solution. Do you actively dislike Derivative?